src/Modules/OrchardCore.Commerce/Drivers/OrderPartDisplayDriver.cs

```diff
@@ -65,49 +66,12 @@ public override async Task<IDisplayResult> UpdateAsync(OrderPart part, IUpdateMo
     private async ValueTask PopulateViewModelAsync(OrderPartViewModel model, OrderPart part)
     {
         model.ContentItem = part.ContentItem;
-        var products = await _productService.GetProductDictionaryAsync(part.LineItems.Select(line => line.ProductSku));
-        var lineItems = await Task.WhenAll(part.LineItems.Select(async lineItem =>
-        {
-            var product = products[lineItem.ProductSku];
-            var metaData = await _contentManager.GetContentItemMetadataAsync(product);
+        var lineItems = part.LineItems;
+        var lineItemViewModelsAndTotal = await OrderLineItemHelpers
+            .CreateOrderLineItemViewModelsAndTotalAsync(lineItems, _contentManager, _productService, _taxProviders);
```

Comment:
Why not make it a service instead of a static helper method? Is there any reason its class can't just own these dependency services you are passing in?